### PR TITLE
Add Novasama test network

### PR DESCRIPTION
**Pull request by stepanLav**
QRs:
scpec:
westend-westend_specs

apng link:
https://spektr-stage-releases.s3.pl-waw.scw.cloud/governance/westend-westend_metadata_1014000.apng?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Content-Sha256=UNSIGNED-PAYLOAD&X-Amz-Credential=SCW70G3H3Z757AXVR08Q%2F20240723%2Fpl-waw%2Fs3%2Faws4_request&X-Amz-Date=20240723T072537Z&X-Amz-Expires=563662&X-Amz-Signature=47aaf2fea76f641e20dd8c216bbcc752680706f9cecb17608b0f9a5c886839f4&X-Amz-SignedHeaders=host&x-id=GetObject

### CODE REVIEW

File: scripts/buildChainsJson.js [hunk 1]
```diff
@@ -95,6 +95,11 @@ function fillAssetData(chain) {
       return;
     }
 
+    // Remove Snowbridge tokens
```
Comment (stepanLav) [inline scripts/buildChainsJson.js:98]: skip snowbridge tokens

File: scripts/buildChainsJson.js [hunk 2]
```diff
@@ -164,7 +169,10 @@ function getTransformedData(rawData) {
     }
 
     const assets = fillAssetData(chain)
-    const nodes = chain.nodes.filter(node => !node.url.includes('{'));
+    const nodes = chain.nodes.filter(node => !node.url.includes('{')).map(node => ({
+      url: node.url,
+      name: node.name
```
Comment (stepanLav) [inline scripts/buildChainsJson.js:174]: keep only url and name parameter for nodes

File: scripts/buildChainsJson.js [hunk 3]
```diff
@@ -188,24 +196,32 @@ function getTransformedData(rawData) {
 
 function replaceUrl(url, type, name = undefined) {
   const changedBaseUrl = url.replace("nova-utils/master", "nova-spektr-utils/main");
-  const lastPartOfUrl = url.split("/").pop()
+  const lastPartOfUrl = url.split("/").pop();
+
+  const customAssetMappings = {
```
Comment (stepanLav) [inline scripts/buildChainsJson.js:201]: custom mapping for particular assets in order to find appropriate icon path

File: scripts/buildChainsJson.js [hunk 4]
```diff
@@ -188,24 +196,32 @@ function getTransformedData(rawData) {
...
+    'aSEEDp': 'aSEED-Polkadot'
+  };
 
   // handling for 'xc' prefixed token names
   const processedName = name ? name.replace(/^xc/, '') : name;
 
   switch (type) {
     case "chain":
+      const chainName = lastPartOfUrl.split('.')[0];
+      const correctedChainName = chainName.charAt(0).toUpperCase() + chainName.slice(1);
```
Comment (stepanLav) [inline scripts/buildChainsJson.js:212]: fix lower case from nova-utils

File: scripts/data/allowedChains.json [hunk 5]
```diff
@@ -20,7 +20,7 @@
   "48239ef607d7928874027a43a67689209727dfb3d3dc5e5b03a39bdc2eda771a": "Kusama Asset Hub",
   "00dcb981df86429de8bbacf9803401f09485366c44efbf53af9ecfab03adc7e5": "Kusama Bridge Hub",
   "c1af4cb4eb3918e5db15086c0cc5ec17fb334f728b7c65dd44bfe1e174ff8b3f": "Kusama People",
-  "d67c91ca75c199ff1ee9555567dfad21b9033165c39977170ec8d3f6c1fa433c": "Ledger Novasama Testnet",
+  "3dbb473ae9b2b77ecf077c03546f0f8670c020e453dddb457da155e6cc7cba42": "Novasama Testnet - Governance",
```
Comment (stepanLav) [inline scripts/data/allowedChains.json:23]: changed genesis

**Review by pgolovkin — APPROVED** [review]

**Pull request merged**